dna-src/minersweeper/index.ts

```diff
@@ -129,15 +129,19 @@ function isDead(gameBoard: GameBoard, actions: Action[]): boolean {
 ============================================*/
 
 // the main validation function of the game. All game rules are enforced here
+// only allow actions to be taken on locations that have not aready been acted on
 function validateAddAction(gameHash, actionHash, agentHash) {
-  let gameBoard = get(gameHash);
+  let action: Action = get(actionHash);
+  if(action.actionType === "chat") return true;
```

Comment:
note to us: maybe we should use some consts for the actionTypes now that we lost the enum...